distributed/deploy/tests/test_local.py

```diff
@@ -1061,3 +1063,17 @@ async def test_cluster_names():
 
         async with LocalCluster(processes=False, asynchronous=True) as unnamed_cluster2:
             assert unnamed_cluster2 != unnamed_cluster
+
+
+@pytest.mark.asyncio
+async def test_local_cluster_redundant_kwarg():
+    with captured_logger("distributed.core", level=logging.WARNING) as log:
+        cluster = await LocalCluster(
+            asynchronous=True, processes=False, typo_kwarg="foo"
+        )
+        client = await Client(cluster, asynchronous=True)
```

Comment:
Could you switch these over to use context managers in order to avoid calling methods like `.close()` later?